driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java

```diff
@@ -128,6 +134,19 @@ public void makeActive() {
         }
     }
 
+    public void makeActive() {
+        makeActiveDb();
+
+        try {
+            this.database.getMetadata().getSchema().reload();
+        } catch (Exception _) {
+            this.database = new ODatabaseDocumentTx(this.database.getURL(), this.database.isKeepStorageOpen());
```

Comment:
what about the pool? don't need to recreate it as well?

just create a new database when pool is active doesn't seem to respect the pool.